fr_CA/home/src/main/resources/rdf/display/firsttime/aboutPage_fr_CA.ttl

```diff
@@ -0,0 +1,23 @@
+# baseURI: http://vivoweb.org/ontology/aboutPage_fr_CA.n3
+
+@prefix : <http://vivoweb.org/ontology/aboutPage_fr_CA.n3#> .
+@prefix about: <http://vitro.mannlib.cornell.edu/ns/default/about#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+about:ABOUTDG
+  a <java:edu.cornell.mannlib.vitro.webapp.utils.dataGetter.FixedHTMLDataGetter> ;
+  <http://vitro.mannlib.cornell.edu/ontologies/display/1.1#htmlValue> """<h2>Über VIVO</h2>
```

Comment:
There seems to be a bit of a mismatch between the intended language of this file and the content.